app/models/katello/repository.rb

```diff
@@ -119,6 +119,7 @@ class Repository < Katello::Model
 
     before_validation :set_pulp_id
     before_validation :set_container_repository_name, :unless => :skip_container_name?
+    before_update :prevent_updates, :unless => :allow_updates?
```

Comment:
This line is absolutely hilarious and I love it :smile:

Reply:
Ruby art